src/wallet/wallet.h

```diff
@@ -900,8 +900,11 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
     /* Set the HD chain model (chain child index counters) */
     bool SetHDChain(const CHDChain& chain, bool memonly);
 
+    /* Generates a new HD master key (will not be activated) */
+    CPubKey GenerateNewHDMasterKey(CKey& key);
```

Comment:
nit: argument is never used anywhere, but I could see this being useful in future